tests/src/test/java/alluxio/shell/command/PersistCommandTest.java

```diff
@@ -67,6 +67,50 @@ public void persistDirectoryTest() throws IOException, AlluxioException {
   }
 
   @Test
+  public void persistMultiFilesTest() throws IOException, AlluxioException {
+    String filePath1 = "/testPersist/testFile1";
+    String filePath2 = "/testFile2";
+    String filePath3 = "/testPersist/testFile3";
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath1, WriteType.MUST_CACHE, 10);
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath2, WriteType.MUST_CACHE, 20);
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath3, WriteType.MUST_CACHE, 30);
+
+    Assert.assertFalse(mFileSystem.getStatus(new AlluxioURI(filePath1)).isPersisted());
+    Assert.assertFalse(mFileSystem.getStatus(new AlluxioURI(filePath2)).isPersisted());
+    Assert.assertFalse(mFileSystem.getStatus(new AlluxioURI(filePath3)).isPersisted());
+
+    int ret = mFsShell.run("persist", filePath1, filePath2, filePath3);
+    Assert.assertEquals(0, ret);
+    checkFilePersisted(new AlluxioURI(filePath1), 10);
+    checkFilePersisted(new AlluxioURI(filePath2), 20);
+    checkFilePersisted(new AlluxioURI(filePath3), 30);
+  }
+
+  @Test
+  public void persistFilesAndDirectoriesTest() throws IOException, AlluxioException {
```

Comment:
Could this method name be more descriptive? Also, it would be good to add a comment for the intention of this test, it is not clear at first sight.